neqo-transport/src/connection/mod.rs

```diff
@@ -1319,6 +1319,9 @@ impl Connection {
         );
         self.saved_datagrams.save(epoch, d, now);
         self.stats.borrow_mut().saved_datagrams += 1;
+        // We already counted the datagram as received in `[input_path]`. We
```

Comment:
```suggestion
        // We already counted the datagram as received in [`input_path`]. We
```

Convention is to put the ticks inside.

https://doc.rust-lang.org/rustdoc/write-documentation/linking-to-items-by-name.html

Note that this will not be picked up by `cargo docs`, given that it is a `//` and not a `///`.